schema/geometry-schema.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://docs.predicthq.com/docs/schema/geometry-schema.json",
+    "title": "GeoJSON Geometry",
+    "oneOf": [
+      {
+        "title": "GeoJSON Point",
+        "type": "object",
+        "required": ["type", "coordinates"],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": ["Point"]
+          },
+          "coordinates": {
+            "type": "array",
+            "minItems": 2,
+            "items": {
+              "type": "number"
+            }
+          },
+          "bbox": {
+            "type": "array",
+            "minItems": 4,
+            "items": {
+              "type": "number"
+            }
+          }
+        }
+      },
+      {
+        "title": "GeoJSON MultiPoint",
```

Comment:
Question 2: This schema is used in events json schema mentioned in the ticket. For events, based on the [doc](https://docs.predicthq.com/api/events/search-events#:~:text=geo-,object,-An%20object%20containing), we only support the Point, Polygon and MultiPolygon. Not sure if I should keep the ones we not supported ts as seeing our old doc kept all types.

Reply:
Hmm, great question, given this is a generic geometry schema and not only specific to the events, I am inclined to  add all possible geometry types as you did. But in the events doc page we make it clear which types are supported.

The other option is to make this file specific to events and rename it to something like `event-geometry-schema.json` and then only add supported types.